test/test-dbPatients.js

```diff
@@ -68,6 +68,28 @@ describe('DB Patients', () => {
           doctorAddress,
           doctorPhone,
         );
+        const exist = await testDataPatientsExist(lastName);
+        exist.should.be.equal(true);
+      } catch( error ) {
+        expect(error).to.be.an('Error');
+      }
+    });
+
+    it('should refuse INsert for INE with more than 50 characters in PG', async () => {
+      const psychologistId = '357a2085-6d32-44db-8fe6-979c7339fd47'
+      try {
+        await dbPatients.insertPatient(
+          firstNames,
+          lastName,
+          "515151515151515151515151551515151515151515151515151",
```

Comment:
Hey, little suggestion for readability here:
```
"1".repeat(51)
```
it won't change much but we don't have to count to ensure that the test should pass or fail.

Reply:
i love it ! thanks